.github/workflows/push.yml

```diff
@@ -423,7 +423,7 @@ jobs:
           TAG: ${{ needs.set-version.outputs.tag }}
         run: |
           hub release download "${TAG}" -i '*windows_amd64*.zip'
-          printf "zip=%s\n" *.zip >> $GITHUB_OUTPUT
+          printf "zip=%s" *.zip >> $GITHUB_OUTPUT
```

Comment:
Myslím si tak naivně, že by se do toho `$GITHUB_OUTPUT` neměla posílat ta newline (tady ji neposílají: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter). Jinak se to chová stejně a jinak mě nenapadá co by mohlo být blbě. 😕 Failuje níže build msi

![image](https://user-images.githubusercontent.com/215660/195893460-28958a4c-cd7b-4b89-af98-369b1070c644.png)